src/Orckestra.Composer.Website/App_Data/Razor/Composer/Cart/Coupons.cshtml

```diff
@@ -48,9 +48,9 @@
 @helper Coupons()
 {
     <div v-if="Coupons">
-        <div v-for="msg in Coupons.Messages" >
+        <div v-if="ShowAlert" v-for="msg in Coupons.Messages" >
             <div v-bind:class="'alert  alert-' + msg.Level + ' alert-dismissible'">
-                <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+                <button type="button" class="close" aria-label="Close" v-on:click="hideAlert()">
```

Comment:
can be simplified v-on:click="ShowAlert = false" without adding the additional method